Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM python:3.9
+FROM python:3.10-slim
 
-ADD Launcher.py .
-ADD requirements.txt .
-ADD localConfig.json .
-ADD bob.txt .
-
-RUN pip install -r requirements.txt 
+COPY Launcher.py requirements.txt localConfig.json bob.txt /
```

Comment:
Files bob.txt and localConfig.json must be writable by the user, we can achieve this with a chmod or a chown.

Here's a suggested change for the first method:
```suggestion
COPY --chmod=777 bob.txt localConfig.json /
COPY Launcher.py requirements.txt  /
```

I let you decide to accept this change or propose something else if you prefer.

I'll let the container running this night and merge your PR tomorrow if everything is ok.

Reply:
Really nice finding! 
Give me some time, both ways are good, but let me know what do you think about my way?
I've added copy with chown to `exorde_user`